app/(main)/settings/team/agent-usage-table.tsx

```diff
@@ -0,0 +1,59 @@
+import { formatTimestamp } from "@/app/(playground)/p/[agentId]/canary/lib/utils";
+
+export type AgentActivity = {
+	agentId: string;
+	agentName: string | null;
+	startTime: Date;
+	endTime: Date | null;
+	usedCharge: number;
+};
+
+type AgentUsageTableProps = {
+	activities: AgentActivity[];
+	containerClassName?: string;
+};
+
+export function AgentUsageTable({
+	activities,
+	containerClassName,
+}: AgentUsageTableProps) {
+	return (
+		<div className="font-avenir rounded-[16px]">
+			<div className="grid grid-cols-4 gap-4 border-b border-zinc-800 bg-zinc-900/50 p-4 font-medium text-zinc-200">
+				<div>Agent</div>
+				<div>Start Time</div>
+				<div>End Time</div>
+				<div>Charge</div>
+			</div>
+			<div className={containerClassName}>
+				<div className="divide-y divide-zinc-800">
+					{activities.length > 0 ? (
+						activities.map((activity) => (
+							<div
+								key={`${activity.agentId}-${activity.startTime}`}
+								className="grid grid-cols-4 gap-4 p-4 items-center text-zinc-200"
+							>
+								<div>{activity.agentName ?? activity.agentId}</div>
+								<div className="text-zinc-400">
```

Comment:
I'd like to display datetime in user's local timezone.
(Since the existing `formatTimestamp` function is primarily used for relative time formatting.)

How about adding a Client Component like this:

```tsx
'use client';

interface LocalDateTimeProps {
  utcDateTime: Date;
  // Optional format configuration
  format?: Intl.DateTimeFormatOptions;
}

export default function LocalDateTime({ 
  utcDateTime, 
  format = {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
    hour: '2-digit',
    minute: '2-digit',
    hour12: false
  }
}: LocalDateTimeProps) {
  // Display using browser's locale and timezone
  return (
    <time dateTime={utcDateTime.toISOString()}>
      {utcDateTime.toLocaleString(undefined, format)}
    </time>
  );
}
```

Reply:
@satococoa [d190b00](https://github.com/giselles-ai/giselle/pull/236/commits/d190b00ecbecab414f59a363697b11a900d49ca2) updated 🙏